apps/prairielearn/src/server.js

```diff
@@ -1904,6 +1904,14 @@ export function initExpress() {
     },
     require('./pages/publicQuestions/publicQuestions').default,
   ]);
+  app.use(
+    '/pl/course/:course_id(\\d+)/cacheableElements/:cachebuster',
```

Comment:
Was this intended to be the public prefix?

Reply:
yes, thanks. I obviously have not finished testing this properly yet (there are other problems I know about too), which is why it is still marked 'draft'.